### PR TITLE
Feat: GFW Assets version control 

### PR DESCRIPTION
**Pull request by geriux**
1- Added the staging environment so we can differentiate between production and staging.
2- Manage GFW Assets version depending on the environment:
* Latest -> Production only
* Nightly -> Staging and dev
* Dev -> Only development

IMPORTANT - We have to add the environment variable on production and staging -> ASSETS_VERSION=latest

### CODE REVIEW

File: app/views/shared/_gfw_assets.html.slim [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- if Rails.env.production?
+  script src="http://gfw-assets.s3.amazonaws.com/static/gfw-assets.latest.js" id="loader-gfw" data-current=".shape-climate"
+- else
+  script src="http://gfw-assets.s3.amazonaws.com/static/gfw-assets.nightly.js" id="loader-gfw" data-current=".shape-climate"
```
Comment (geriux) [inline app/views/shared/_gfw_assets.html.slim]: @davidsingal Hey! Any thoughts about this?? =D

File: config/environments/staging.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
+Rails.application.configure do
```
Comment (geriux) [inline config/environments/staging.rb]: @simaob So I added this to the config environments folder so we can differentiate between production and staging. Is this ok? Or do I have to add something else?

File: app/views/shared/_gfw_assets.html.slim (hunk 1 above)
Comment (davidsingal) [inline app/views/shared/_gfw_assets.html.slim]: I prefer something like this:
`assets_version= Rails.env.production? 'latest' : 'nightly'`
but better is:
in dot env file add `ASSETS_VERSION=latest` and use this like `${ENV['ASSETS_VERSION']}`

Comment (geriux) [inline app/views/shared/_gfw_assets.html.slim]: Way better with an env variable, thanks!! 😁

File: config/environments/staging.rb (hunk 2 above)
Comment (simaob) [inline config/environments/staging.rb]: So we need to make sure that the staging site is behaving in the staging environment.

Comment (simaob) [inline config/environments/staging.rb]: Otherwise it won't use this env config.

**Pull request closed without merging**